osf/management/commands/rebrand_preprint_provider.py

```diff
@@ -0,0 +1,108 @@
+"""
+This script is destructive and should be used with caution.
+Before use, a new provider must already be created with correct subjects and assets.
+
```

Comment:
How is the new provider going to be created? Are we basically copying the old one then adding the subjects/assets/whichever fields need to be updated for the new brand?  Do we need to enforce that all the correct relationships are there?

Reply:
With the "Import/Export JSON" admin app functionality. The `assert` calls in the `rebrand` method are meant to ensure field consistency where required